### DISCUSSION TITLE
Detect SI Units in a language

### DISCUSSION THREAD

**Discussion by PhilipPartsch** [explosion/spaCy#13069] [Help: Best practices]
Hello together,
I'm new to the topic NLP and spaCy.
I currently try to detect SI Units within a text.
Does anybody know how to do this?

**Comment by svlandeg** [comment]
Hi, welcome!

Considering that you're new to spaCy & NLP, I would very much recommend this [free, open-source course](https://course.spacy.io/en/) by Ines, which starts from the foundation, builds things up interactively, and has many useful tips and tricks to help you get started. It will be well worth your time to better understand how things work :-)

Then after that, I recommend having a read through our documentation on [pattern-based matching](https://spacy.io/usage/rule-based-matching), which will probably be sufficient to solve your SI Units use-case.
